projects/elife.yaml

```diff
@@ -1057,7 +1057,6 @@ elife-dashboard:
 elife-reporting:
     formula-repo: https://github.com/elifesciences/elife-reporting-formula
     vagrant:
-        box: ubuntu/trusty64 # Ubuntu 14.04, deprecated
```

Comment:
this is unrelated to the intent of this PR but I forgot to switch branches before commit